backend/users/api/views.py

```diff
@@ -1,16 +1,92 @@
+from email import message
 from rest_framework import exceptions, status
 from rest_framework.views import APIView
 from rest_framework.response import Response
+from rest_framework.exceptions import AuthenticationFailed
+from django.contrib.auth.hashers import make_password
+import jwt, datetime
 
 from .serializers import UserSerializer
+from users.models import User
+
+JWT_SECRET = "sels-project"
 
 class RegisterAPI(APIView):
 
     def post(self, request):
+
+        # hash the password
+        hashedPassword = make_password(request.data["password"])
+        request.data["password"] = hashedPassword
+        print(request.data["password"])
```

Comment:
- dont forget that we are removing console.log and print 
- we dont typically use comments for simple logics. variable, functions, and classnames should be self explanatory that can explain what you did in this block of logic.If you use comments on simple logics, meaning your code is not readable.